src/test/MethodCodeTest.java

```diff
@@ -75,9 +75,9 @@ void checkGetMethodOutput() throws Throwable {
 //        MethodUtil.invokeIfMethodExists(MethodCode.class, "ThisMethod");
 //    }
 
-    @Test
```

Comment:
Why is this commented out?

Reply:
Probably because I was just testing something and I just wanted to enable you and Opey to be able to review the source MethodTest code

Reply:
Should we change it back now?

Reply:
It was because I was just using them to make sure errors were being thrown when they should